assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -127,15 +98,17 @@ header[role="banner"] {
   background-color: gu-colour(header-dark);
   border-bottom: 1px solid gu-colour(garnett-neutral-4);
   position: relative;
-  height: 75px;
```

Comment:
could this be inferred from the height of the contents (`.glogo`) and the padding instead of explicitly set?